documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -29,6 +29,17 @@ You can use {the-lc} {project-first} to migrate virtual machines from the follow
 * Open Virtual Appliances (OVAs) that were created by VMware vSphere
 * Remote {virt} clusters
 
+Migration using Open Virtual Appliance (OVA) files as source providers is a Technology Preview.
+
+[IMPORTANT]
+====
+Migration using Open Virtual Applliance (OVA) files as source providers is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
```

Comment:
this part is repeating 3 times in this PR. worth pulling it out to another adoc file?